<launch>
<node pkg="habitat_recognition" type="main.py" name="habitat_recognition_node" output="screen">
  <!-- <param name="agent_type" value="random_walk"/> -->
  <!-- <param name="rgb_topic" value="/camera/rgb/image"/> -->
  <!-- <param name="depth_topic" value="/camera/depth/image"/> -->
  <!-- <param name="camera_info_topic" value="/camera/rgb/camera_info"/> -->
  <!-- <param name="true_pose_topic" value="/camera/true_pose"/> -->
  <param name="cloud_topic" value="/rtabmap/cloud_map"/>
  <param name="camera_calib" value="/home/junting/habitat_ws/src/habitat_ros/camera_info.yaml"/>
  <param name="test_scene" value="/media/junting/SSD_data/habitat_data/scene_datasets/mp3d/v1/scans/17DRP5sb8fy/17DRP5sb8fy.glb"/>
</node>
</launch>
